--- imap-2004d/src/osdep/unix/Makefile.optflags	2005-04-29 12:53:09.000000000 -0500
+++ imap-2004d/src/osdep/unix/Makefile	2005-04-29 12:55:16.000000000 -0500
@@ -36,7 +36,7 @@
 # -O3 seems to be safe, but empirical observation from our local expert
 # indicates that in some (many?) cases -O3 code runs slower than -O2.
 
-GCCOPTLEVEL= -O2
+GCCOPTLEVEL= -O2 $(RPM_OPT_FLAGS)
 
 
 # Extended flags needed for SSL.  You may need to modify.
@@ -454,7 +454,7 @@
 	 SPOOLDIR=/var/spool \
 	 ACTIVEFILE=/var/lib/news/active \
 	 RSHPATH=/usr/bin/rsh \
-	 BASECFLAGS="-g -fno-omit-frame-pointer $(GCCOPTLEVEL)" \
+	 BASECFLAGS="$(GCCOPTLEVEL)" \
 	 BASELDFLAGS="$(PAMLDFLAGS)"
 
 lnx:	# Linux non-shadow passwords
